exec/java-exec/src/test/resources/json/ctas_alltypes_map_out.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
minor point: in TestJsonReader lately we were embedding the data generation in the code instead of creating new files..but for small files like these I suppose this is fine.